ext/nnstreamer/tensor_filter/tensor_filter_caffe2.cc

```diff
@@ -559,32 +559,35 @@ caffe2_getOutputDim (const GstTensorFilterProperties * prop,
  * @param[in] data The data element.
  */
 static void
-caffe2_destroyNotify (void *data)
+caffe2_destroyNotify (void **private_data, void *data)
 {
   /* do nothing */
 }
 
 static gchar filter_subplugin_caffe2[] = "caffe2";
 
 static GstTensorFilterFramework NNS_support_caffe2 = {
-  .name = filter_subplugin_caffe2,
-  .allow_in_place = FALSE,      /** @todo: support this to optimize performance later. */
-  .allocate_in_invoke = TRUE,
-  .run_without_model = FALSE,
-  .verify_model_path = FALSE,
-  .invoke_NN = caffe2_run,
-  .getInputDimension = caffe2_getInputDim,
-  .getOutputDimension = caffe2_getOutputDim,
-  .setInputDimension = NULL,
+  .version = GST_TENSOR_FILTER_FRAMEWORK_V0,
   .open = caffe2_open,
   .close = caffe2_close,
-  .destroyNotify = caffe2_destroyNotify,
+  {
+    .name = filter_subplugin_caffe2,
+  }
```

Comment:
Doesn't this trigger error/warning with SVACE/Coverity?

Reply:
Moved `name` assignment to inside the `ini_filter_*`.